### PR TITLE
Job offer

### PR DESCRIPTION
**Pull request by sauloperez**
Ths adds a new simple layout where the brand image doesn't stand out so much and adds the content for the 1st job offer.

job

### CODE REVIEW

File: _layouts/simple.html [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+       </ul>
+    </nav>
+    <div class="container-fluid">
+       <div class="row">
+          <div class='mx-auto col-12 col-sm-10 col-md-8 content'>
+
+             {{ content }}
+             {% include footer.html %}
+             {% include google_analytics.html %}
+  </body>
```
Comment (enricostano) [inline _layouts/simple.html]: 2 `</div>` tags missing, right?

File: jobs.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
+---
+layout: simple 
+---
+
+# Desenvolupador backend
```
Comment (enricostano) [inline jobs.md]: Can we adopt a more gender neutral way to express this? `Desenvolupador/a`?

File: jobs.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+## Sobre Coopdevs
+
+Coodevs és una entitat que pretén aportar software lliure a l'economia social i solidària. Sorgida des de professionals del sector tecnològic de Barcelona amb inquietuds socials, proporciona amplis coneixements tècnics i metodologies de treball en aquest àmbit econòmic. Entre molts altres projectes, impulsem Katuma, formem part de l'equip de desenvolupament d'Open Food Network, desenvolupem TimeOverflow i tenim a projectes cooperatius
```
Comment (enricostano) [inline jobs.md]: `s/Coodevs/Coopdevs/`

Link to Katuma, OFN and TO?

`i tenim a projectes cooperatius`, I don't understand this

File: jobs.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+## Sobre la feina
+
+Ara busquem gent amb esperit cooperatiu que tingui ganes de compartir aquest repte aportant la seva experiència de desenvolupament de software en un moment important per fer créixer el projecte.
+
+El candidat s'unirà principalment a l'equip que treballa en la línia de negoci d'ERP per a cooperatives que es desenvolupa en Python. Aquesta unitat s'encarrega d'acompanyar i ajudar a cooperatives i altres entitats a la sistematització dels seus processos interns i la seva optimització amb software ERP. Les tasques són:
```
Comment (enricostano) [inline jobs.md]: `El/la`?

File: jobs.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+
+* Implementacions d'ERP a empreses
+* Integracions entre ERP i serveis externs
+* Acompanyament a altres membres junior de l'equip
+* Desenvolupament de backend web
+
+A més, donada la rotació de tasques i diversitat dels projectes que tenim fa que la tasca no es limiti en aquest àmbit i també es puguin dedicar hores als projectes de Katuma, TimeOverflow o projectes específics per altres clients, sempre en l'àmbit web.
+
+Busquem doncs, algú que compleixi els següents criteris:
```
Comment (enricostano) [inline jobs.md]: `algu/alguna`? :thinking:

File: jobs.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+
+* Experiència i bases sòlides en disseny i programació orientada a objectes
+* Haver desenvolupat aplicacions web amb frameworks open source en Python
+    o Ruby tals com Django, Flask, Ruby on Rails, Sinatra, etc.
+* Experiència i ús habitual de tests unitaris i d'integració
+* Coneixement de git suficient per gestionar tots els aspectes d'un repositori
```
Comment (enricostano) [inline jobs.md]: `s/git/Git`

**Review by enricostano — CHANGES_REQUESTED** [review]
Could we add some kind of invitation like "Join us and let's impact on SSE together!"... or something like that.

Or something funny, I don't know...

File: jobs.md (hunk 3 above)
Comment (sseerrggii) [inline jobs.md]: ```Coodevs és una entitat que aporta software lliure```

Comment (sseerrggii) [reply jobs.md]: tenim altres projectes cooperatius?

Comment (sauloperez) [reply jobs.md]: Em referia a altres clients però ho he tret i menys lio

**Comment by sauloperez** [comment]
## Missing

- [x] have a unique URL for this offer so we can have `/jobs` as index.

File: jobs.md (hunk 3 above)
Comment (sseerrggii) [reply jobs.md]: millor parlar en present que no dir "preten aportar"

**Comment by sauloperez** [comment]
Last version

jobs

job

**Pull request merged**